### DISCUSSION TITLE
Zusätzlichen / alten Klingelknopf anschließen

### DISCUSSION THREAD

**Discussion by heckmic** [frickelzeugs/FingerprintDoorbell#80] [Ideas]
Hallo,

bei Regen löst es die klingel immer wieder mal aus oder geht wegen der Nässe nicht.
Ich würde gerne zusätzlich meinen alten Klingelknopf an den ESP anschließen und als einfaches Klingelsignal verwenden.

**Comment by mickler2021** [comment]
Hi, das wäre doch nur eine einfache Verkabelung. Parallel zum Relais, welches vom ESP ausgelöst wird, um die Klingel zum Läuten zu bringen, verdrahtest Du den alten Klingelknopf. Der Klingelknopf muss nicht an den ESP für Dein Vorhaben, falls es nur um das Klingeln geht.

**Reply by heckmic** [reply]
Die Klingel ist bei mir nicht damit verdrahtet sondern wird via iobroker ausgelöst. Das Drücken des Buttons würde ich via MQTT mitbekommen wollen um dann die Funkklingel auszulösen.
